extensions/2.0/Vendor/GRIFFEL_bim_data/README.md

```diff
@@ -0,0 +1,208 @@
+# GRIFFEL\_bim\_data
+
+## Contributors
+
+* Kiryl Ambrazheichyk, Griffel Studio LLC, [kiryl@griffelstudio.com](mailto:kiryl@griffelstudio.com)
+* Aliaksandr Valodzin, Griffel Studio LLC, [alexander@griffelstudio.com](mailto:alexander@griffelstudio.com)
+* Aliaksandr Nazarau, Griffel Studio LLC
+* Ekaterina Kudelina, Griffel Studio LLC, [@uncio](https://github.com/uncio)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows keeping domain specific data in relation to geometry.
+
+It's applicable when glTF standard `extras` property is not enough to store huge amount of metadata or when it's insufficient from the file size or data complexity perspective.
+
+The extension was originally developed for architecture, engineering and construction (AEC) industry to keep Building Information Model (*BIM*) data, but it's suitable for any other industry which operates huge amouts of information. We encourage any subject matter experts to discuss and improve this specification to create useful general metadata glTF extension.
```

Comment:
Please clarify the intention here.

A vendor-specific extension spec may be provided as-is purely for interoperability and preservation reasons. Alternatively, a vendor-specific extension spec may serve as a starting point for further development and collaboration. In the latter case, the vendor-specific extension may become a multi-vendor extension (with `EXT_` prefix) if there's enough interest.

It looks like there are shipping products that use this extension already so any further development may require a new extension.

Reply:
This is a vendor specific extension. For now it's going to be used in one of our products. If others will use it and implement on their sides they might want to introduce some improvements, might want to collaborate on the specification. In that case we are ready to change our implementation. As only one small product implements it and we are going to support any changes, no new extension will be needed.